cms_helper.py

```diff
@@ -5,11 +5,6 @@
 HELPER_SETTINGS = dict(
     ROOT_URLCONF="tests.example_app.urls",
     INSTALLED_APPS=["meta", "tests.example_app"],
-    META_SITE_PROTOCOL="http",
-    META_USE_SITES=True,
-    META_USE_OG_PROPERTIES=True,
-    META_USE_TWITTER_PROPERTIES=True,
-    META_USE_SCHEMAORG_PROPERTIES=True,
```

Comment:
these are meant to be used when running `app_helper.py server` to run the sample project

Reply:
Oh, ok, I see... if I leave them there, some tests are failing somehow because I think tests use those settings. 

For instance, `test_meta.py::MetaObjectTestCase::test_defaults` fails because `self.assertEqual(m.use_og, False)` is actually `True`, because of `META_USE_OG_PROPERTIES=True` in `cms_helper.py`

What's the best approach here? Override those settings globally in every test class, in order to reflect the actual default defined in `settings.py`?

Reply:
I see

I can't think of a clean solution, so maybe the cleanest is to use `override_settings` on `BaseTestClass` to ensure all the tests use the desired setting no matter what `cms_helper` sets?

Reply:
Done!